### ISSUE TITLE
Update Spanish translations

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/infrastructure#467]
About 50% translated in 0.9.4.

**Comment by duncandewhurst** [comment]
The new pages in 0.9.4 were missing from the list of resources in Transifex so I followed the [steps](https://ocds-standard-development-handbook.readthedocs.io/en/latest/standard/translation/technical.html) in the handbook to extract, map and push strings to Transifex.

After doing so, the translation is more like ~30% complete.

**Comment by duncandewhurst** [comment]
@EvelynDinora, regarding the [sustainability modules documentation](https://standard.open-contracting.org/staging/infrastructure/0.9-dev/en/cost/ids/sustainability/), please confirm which parts you want to be translated to Spanish:

![sustainability_mapping](https://github.com/open-contracting/infrastructure/assets/19265814/3158976d-d7a2-433a-8934-c7b11489574a)

**Comment by duncandewhurst** [comment]
I've taken the OCDS Development Handbook instructions for translators and I'm adapting them for OC4IDS in a [Google Doc](https://docs.google.com/document/d/1y6EMJZOCaPiGnWr0QT2KipCtVh1W1MOHVIJNrigiDss/edit#heading=h.ao3xzfkkkltz).

If the mappings need to be translated, I think we'll need to add some extra instructions on which parts to leave untranslated, e.g. don't translate words in backticks or single quotes. Likewise, we should explain what to do with relative URLs, e.g. if a heading in the schema reference Markdown page is translated, should the translator update the references to that heading, or is that something that we do afterwards as part of a tidy-up?

**Comment by jpmckinney** [comment]
Is there a reason there is a copy for OC4IDS, rather than making changes to the original? (e.g. moving OCDS-specific content to admonitions). From the doc's change history, it looks like it's mostly changing the examples to be about OC4IDS.

**Comment by duncandewhurst** [comment]
Mostly that all the links pointed to the OCDS docs and OCDS Transifex project and I think it'll be easier for the translator to follow the instructions if the links point to the project that the translator is working on. Happy to discuss how best to integrate the docs for both projects, this was just a quick route to having something the OC4IDS translator can work with.

**Comment by jpmckinney** [comment]
Cool, we can just provide both links.

**Comment by duncandewhurst** [comment]
Like this? (Just checking before I add them all)

> 1. Go to the list of resources ([OCDS](), [OC4IDS]())

**Comment by jpmckinney** [comment]
Yup, that works.

**Comment by EvelynDinora** [comment]
> @EvelynDinora, regarding the [sustainability modules documentation](https://standard.open-contracting.org/staging/infrastructure/0.9-dev/en/cost/ids/sustainability/), please confirm which parts you want to be translated to Spanish:
> 
> ![sustainability_mapping](https://private-user-images.githubusercontent.com/19265814/308714689-3158976d-d7a2-433a-8934-c7b11489574a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk2NTAwOTMsIm5iZiI6MTcwOTY0OTc5MywicGF0aCI6Ii8xOTI2NTgxNC8zMDg3MTQ2ODktMzE1ODk3NmQtZDdhMi00MzNhLTg5MzQtYzdiMTE0ODk1NzRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA1VDE0NDMxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRkYmEwYzExYTk2NTNlN2MyMzEwZThhY2ZjYmIwNzMwNTVhMjFiZThmYTkzNDY4NzcyMmY0NmM3ZGNmNDk2NWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WaAo_SJwD1rtlcGqpJ8nBnX8nDFRyS_iIpizyvSC7V8)

Hi @duncandewhurst I am happy to translate the updates in the same way and to the same extend we did for the original OC4IDS, please. Many thanks.

**Comment by duncandewhurst** [comment]
@EvelynDinora in the original mapping of the core CoST IDS elements, only the OC4IDS mapping column is translated (see below). Do you want to take the same approach for the sustainability modules, i.e. translate only the OC4IDS mapping and not the CoST IDS element titles and disclosure formats?

![image](https://github.com/open-contracting/infrastructure/assets/19265814/0254dc5b-4b02-421f-b7fd-f8210eb4eb37)

I think it'll be most useful to readers if everything is translated, but I'm not sure if you have already arranged the translation of the CoST IDS elements elsewhere. If so, we can reuse those translations to avoid duplicating work.

**Comment by EvelynDinora** [comment]
> @EvelynDinora in the original mapping of the core CoST IDS elements, only the OC4IDS mapping column is translated (see below). Do you want to take the same approach for the sustainability modules, i.e. translate only the OC4IDS mapping and not the CoST IDS element titles and disclosure formats?
> 
> ![image](https://private-user-images.githubusercontent.com/19265814/310723828-0254dc5b-4b02-421f-b7fd-f8210eb4eb37.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk3OTQ3MDgsIm5iZiI6MTcwOTc5NDQwOCwicGF0aCI6Ii8xOTI2NTgxNC8zMTA3MjM4MjgtMDI1NGRjNWItNGIwMi00MjFmLWI3ZmQtZjgyMTBlYjRlYjM3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA3VDA2NTMyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxZTlhYzU3YTczNzY0ZWFmMTQwZDg4NmRhNWJjZDU0ZmFhNDZhMjlhZDZkNzJlZWJhNTI3YzQ5YjgzMWYzNjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.5-D_k1YHmyzyZuJfa3rldBMrnE7LGb7QCgVxKragBe4)
> 
> I think it'll be most useful to readers if everything is translated, but I'm not sure if you have already arranged the translation of the CoST IDS elements elsewhere. If so, we can reuse those translations to avoid duplicating work.

@duncandewhurst yes, we can translate everything to facilitate what is most useful for the readers, even the original CoST elements that were not translated at the beginning. Many thanks.

**Comment by duncandewhurst** [comment]
@EvelynDinora. Great. I've pushed the missing strings to Transifex. Everything is now ready for the translator.
